### ISSUE TITLE
Success/Jemison/Hollywood Reno Failure.

### ISSUE THREAD

**Issue by jbrz** [mtgred/netrunner#2715]
`Project Atlas` token > get `Biotic` > Install `Hollywood Reno` > `Biotic Labor` >`Success` sacrificing Project Atlas. > Error:
Didn't offer me option to place advancements with `Jemison`, but instead asked immediately for `Success` target; error occurred, and _then_ was prompted for `Jemison` advancements, but appeared to not fire Success, as I wasn't asked for hollywood reno advancement targets.  Only ended up with 3 advancement counters on the agenda.

Stack Trace:
```clojure
Error do select Hollywood Renovation #error {
 :cause nil
 :via
 [{:type java.lang.NullPointerException
   :message nil
   :at [clojure.lang.RT longCast "RT.java" 1241]}]
 :trace
 [[clojure.lang.RT longCast "RT.java" 1241]
  [game.core$fn__23853$fn__23859 invoke "operations.clj" 1294]
  [game.core$do_effect invokeStatic "abilities.clj" 260]
  [game.core$do_effect invoke "abilities.clj" 256]
  [game.core$do_ability invokeStatic "abilities.clj" 241]
  [game.core$do_ability invoke "abilities.clj" 214]
  [game.core$resolve_ability_eid invokeStatic "abilities.clj" 136]
  [game.core$resolve_ability_eid invoke "abilities.clj" 113]
  [game.core$resolve_ability invokeStatic "abilities.clj" 111]
  [game.core$resolve_ability invoke "abilities.clj" 38]
  [game.core$resolve_ability invokeStatic "abilities.clj" 109]
  [game.core$resolve_ability invoke "abilities.clj" 38]
  [game.core$resolve_select invokeStatic "abilities.clj" 386]
  [game.core$resolve_select invoke "abilities.clj" 374]
  [game.core$select invokeStatic "actions.clj" 170]
  [game.core$select invoke "actions.clj" 155]
  [game.main$handle_do invokeStatic "main.clj" 77]
  [game.main$handle_do invoke "main.clj" 74]
  [game.main$handle_command invokeStatic "main.clj" 145]
  [game.main$handle_command invoke "main.clj" 134]
  [game.main$run$fn__32138 invoke "main.clj" 180]
  [game.main$run invokeStatic "main.clj" 170]
  [game.main$run invoke "main.clj" 164]
  [game.main$_main$fn__32173 invoke "main.clj" 234]
  [clojure.lang.AFn run "AFn.java" 22]
  [java.lang.Thread run "Thread.java" 745]]}
```

**Comment by JoelCFC25** [comment]
Duplicate of #2704.

**Comment by JoelCFC25** [comment]
Fix coming in #2808.
